packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -391,6 +392,20 @@ export function getStylesDeclarations(
 		[]
 	);
 
+	/*
+	 * Set background defaults.
+	 * Applies to all background styles except the top-level site background.
+	 */
+	if ( ! isRoot && !! blockStyles.background ) {
+		blockStyles = {
+			...blockStyles,
+			background: {
+				...blockStyles.background,
+				...setBackgroundStyleDefaults( blockStyles.background ),
```

Comment:
Another approach happening in https://github.com/WordPress/gutenberg/pull/60578/files#diff-52acb55f718dce035b748387f60ef068cd3769674b553584f7df6f790e9fa84eR403

Reply:
Just double-checking, this comment is no longer relevant, is it? Looks like the approach here is that we're only applying the defaults to block-level in global styles 👍

Reply:
Yep, you're right. It was me cross linking and basically tying string to my fingers so I didn't forget 😄